### PR TITLE
cmd: fix imports order (according to gci)

### PR DESCRIPTION
**Pull request by pedronis**
gci needed cmd/snap-seccomp/main.go import comment to made one long line otherwise it keeps only the last line :(

cmd/snap/color_test.go needed some manual cleanup of commented out imports

### CODE REVIEW

File: cmd/snap-seccomp/main.go [hunk 1]
```diff
@@ -180,9 +180,7 @@ import (
 	"strings"
 	"syscall"
 
-	// FIXME: we want github.com/seccomp/libseccomp-golang but that
-	// will not work with trusty because libseccomp-golang checks
-	// for the seccomp version and errors if it find one < 2.2.0
+	// FIXME: we want github.com/seccomp/libseccomp-golang but that will not work with trusty because libseccomp-golang checks for the seccomp version and errors if it find one < 2.2.0
```
Comment (mardy) [inline cmd/snap-seccomp/main.go:183]: I liked this better as it was before :-)

**Review by mardy — COMMENTED** [review]
All good, I'm just a bit puzzled by a reformatting of a comment.

File: cmd/snap-seccomp/main.go (hunk 1 above)
Comment (pedronis) [reply cmd/snap-seccomp/main.go:183]: as I tried to explain in the description gci  breaks the split one by keeping only the last line

**Review by mvo5 — APPROVED** [review]
Thank you

**Review by mardy — APPROVED** [review]

**Pull request merged**